commit 13ab10c262011036108d3ef24e6ca9ce5b0492e2
Author: Rosalie Wanders <rosalie@mailbox.org>
Date:   Tue Nov 11 11:55:11 2025 +0100

    3rdParty: don't enable asserts when using RelWithDebInfo

diff --git a/Source/3rdParty/CMakeLists.txt b/Source/3rdParty/CMakeLists.txt
index c85d6b70b..f3020f2fa 100644
--- a/Source/3rdParty/CMakeLists.txt
+++ b/Source/3rdParty/CMakeLists.txt
@@ -47,8 +47,10 @@ else()
     set(MAKE_OPTFLAGS "${CMAKE_C_FLAGS}")
 endif()
 
-if ("${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo" OR
-    "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
+if ("${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo")
+    set(MAKE_DEBUG "0")
+    set(MAKE_OPTFLAGS "${MAKE_OPTFLAGS} -g")
+elseif ("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
     set(MAKE_DEBUG "1")
 else()
     set(MAKE_DEBUG "0")
